build/vars.build.mk

```diff
@@ -14,15 +13,14 @@ LOCAL_AM_CFLAGS = \
 	-DDOCDIR=\""$(docdir)"\" \
 	-DGEANYPLUGINS_DATADIR=\""$(datadir)"\" \
 	-DPKGDATADIR=\""$(pkgdatadir)"\" \
-	-DLIBDIR=\""$(libdir)"\" \
-	-DPKGLIBDIR=\""$(pkglibdir)"\"
```

Comment:
I have no idea what it would be on Windows as I also don't know what it might be used for on non-Windows.
Do we want to keep it? I couldn't find any reference to it.